js/algoliasearch/click_conversion_analytics.js

```diff
@@ -1,89 +1,108 @@
 algoliaBundle.$(function ($) {
 	AlgoliaAnalytics.init({
-		applicationID: algoliaConfig.applicationId,
+		appId: algoliaConfig.applicationId,
 		apiKey: algoliaConfig.instant.apiKey
 	});
 
 	// "Click" in autocomplete
 	$(algoliaConfig.autocomplete.selector).each(function () {
-		$(this).on('autocomplete:selected', function (e, suggestion) {
-			trackClick(suggestion.objectID, suggestion.__position, suggestion.__queryID);
+		$(this).on('autocomplete:selected', function (e, suggestion, dataset, context) {
+
+			var sources = analyticsHelper.sources;
+			var source = sources.find(function(e) {
+				return e.name == dataset;
```

Comment:
I'm not sure about this line.
dataset is an integer, so it should be something like `sources[dataset -1]`